src/ado2gh/Commands/AddTeamToRepoCommandHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace OctoshiftCLI.AdoToGithub.Commands
```

Comment:
wouldn't it be better to put the handlers in a separate directory? e.g. `OctoshiftCLI.AdoToGithub.Handlers`

Reply:
agreed, I'll do that